spring-core/src/main/java/org/springframework/cglib/transform/impl/UndeclaredThrowableTransformer.java

```diff
@@ -55,10 +55,20 @@ public CodeEmitter begin_method(int access, final Signature sig, final Type[] ex
             return e;
         }
         return new CodeEmitter(e) {
-            private Block handler;
-            /* init */ {
-                handler = begin_block();
-            }
+	        private final boolean isConstructor = Constants.CONSTRUCTOR_NAME.equals(sig.getName());
+            private Block handler = begin_block();
+	        private boolean calToSuperWasSeen;
```

Comment:
typo perhaps in `cal`?

Reply:
I'm preparing a revision of this to separate `ClassLoaderAwareGeneratorStrategy` from an `UndeclaredThrowableStrategy` delegate, to be committed soon. Noticed this only already, renaming it to `callToSuperSeen`.